### ISSUE TITLE
activesupport dependency

### ISSUE THREAD

**Issue by International** [bear-metal/tunemygc#4]
Hello!

I get the following when trying to add to a rails 3.2.13 app:

```
Fetching source index from http://rubygems.org/
Resolving dependencies.........................................................
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    tunemygc (>= 0) ruby depends on
      activesupport (~> 4.1) ruby

    rails (>= 0) ruby depends on
      activesupport (3.2.13)
```

Could the dependency be changed to support rails 3 apps as well?

**Comment by methodmissing** [comment]
@International sure, will spawn a rails3 branch towards the end of the day / tomorrow morning and ping you for testing

**Comment by International** [comment]
Awesome! Thanks!

**Comment by methodmissing** [comment]
Just pushed 1.0.25 - ActiveSupport is now a dev dependency only. Mind taking it for a spin in your Ralis 3 env?

**Comment by International** [comment]
Thanks for this! The gem installation works, but I get the following error now:

```
undefined method `call' for #<TuneMyGc::StartRequestSubscriber:0x00000002566510>

/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activesupport/lib/active_support/notifications/fanout.rb:47:in `publish'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activesupport/lib/active_support/notifications/fanout.rb:25:in `block in publish'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activesupport/lib/active_support/notifications/fanout.rb:25:in `each'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activesupport/lib/active_support/notifications/fanout.rb:25:in `publish'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activesupport/lib/active_support/notifications/instrumenter.rb:25:in `instrument'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activesupport/lib/active_support/notifications.rb:123:in `instrument'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_controller/metal/instrumentation.rb:27:in `process_action'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_controller/metal/params_wrapper.rb:207:in `process_action'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/activerecord/lib/active_record/railties/controller_runtime.rb:18:in `process_action'
newrelic_rpm (3.9.9.275) lib/new_relic/agent/instrumentation/rails3/action_controller.rb:32:in `block in process_action'
newrelic_rpm (3.9.9.275) lib/new_relic/agent/instrumentation/controller_instrumentation.rb:366:in `perform_action_with_newrelic_trace'
newrelic_rpm (3.9.9.275) lib/new_relic/agent/instrumentation/rails3/action_controller.rb:31:in `process_action'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/abstract_controller/base.rb:121:in `process'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/abstract_controller/rendering.rb:45:in `process'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rack-mini-profiler-7a56508b6548/lib/mini_profiler/profiling_methods.rb:106:in `block in profile_method'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_controller/metal.rb:203:in `dispatch'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_controller/metal/rack_delegation.rb:14:in `dispatch'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_controller/metal.rb:246:in `block in action'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_dispatch/routing/route_set.rb:73:in `call'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_dispatch/routing/route_set.rb:73:in `dispatch'
/home/geo/.rvm/gems/ruby-2.1.5@mt/bundler/gems/rails3/actionpack/lib/action_dispatch/routing/route_set.rb:36:in `call'
journey (1.0.4) lib/journey/router.rb:68:in `block in call'
```

**Comment by methodmissing** [comment]
Ok, looks like a diff between ActiveSupport instrumentation in Rails 3 and 4. Will update shortly

**Comment by International** [comment]
Do you want me to open a new bug and close this one?

**Comment by methodmissing** [comment]
Lets handle it here for context. Ideally Travis should run a 3.x sandbox too, but we're still validating the agent and concept and keeping things lean.

**Comment by methodmissing** [comment]
Pushed https://github.com/bear-metal/tunemygc/commit/77e804f98414c4c721ab08056b24c1c7f7499e7e and released 1.0.28 . Mind giving it a go and closing if you're unblocked?

**Comment by International** [comment]
Works! Thank you!
